src/components/switcher/switcher-styles.module.scss

```diff
@@ -0,0 +1,36 @@
+@import 'styles/settings';
+@import 'styles/ui.module';
+
+$width: 340px;
+$height: 38px;
+$half-width: 170px;
+
+.switcher {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0%;
+  left: calc(50% - #{$half-width});
+  width: 340px;
+  height: 38px;
+  clip-path: polygon(0 0, 100% 0, 88% 100%, 14% 100%);
+  box-shadow: 0 2px 10px 0 rgba(23,27,48,0.15);
+  background-color: $elephant;
+  color: $white;
+  letter-spacing: 1.5px;
+  font-size: $font-size-xxs;
```

Comment:
Please use here `%subtitle`, one of the placeholder Selectors from typography-extends.scss

Reply:
👍